site/2016/07/27/doctrine-mongodb-odm-release-1.1.1.rst

```diff
@@ -0,0 +1,40 @@
+Doctrine MongoDB ODM release 1.1.1
+==================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.1.1 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.1>`__.
+
+Bug fixes in this release
+-------------------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/master/CHANGELOG-1.1.md#111-2016-07-27>`__.
+A full list of issues and pull requests included in this release may be found in the
+`1.1.1 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.1.1>`__.
+
+Installation
+------------
+
+You can install the latest version using the following ``composer.json`` definitions:
+
+.. code-block:: json
+
+  {
+      "require": {
+          "doctrine/mongodb-odm": "^1.1.1"
+      }
+  }
+
+Future Releases
+---------------
+
+Work on much anticipated 2.0 version of ODM supporting new MongoDB driver directly is commencing

```

Comment:
"Work on the much anticipated 2.0 version of MongoDB ODM with support for the new MongoDB driver is beginning, however we are unable to set a release date yet."